repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl

```diff
@@ -83,7 +83,7 @@
                 </nav>
                 <#if document.relatedContent??>
                     <div class="nhsuk-width-container">
-                        <@hee.contentCards contentCards=document.relatedContent/>
+                        <@hee.contentCards contentCards=document.relatedContent relatedContent=true />
```

Comment:
Cookies link is not visible . It works fine now.
Can you check on Related Content on mini hubs , please ? It is still visible when I tried on below page with selecting all headers ( h2,h3,h4) under Anchor links component.  Is it acceptable behaviour ?
http://localhost:8080/site/patient-and-public-information/introduction

![image](https://user-images.githubusercontent.com/95379560/151701443-b3e3da55-9722-4f65-b3fe-ef0fa1bab2ec.png)

Reply:
Hi @sankar-shunmuga - as discussed this should only be applied to the Minihub